docs/userguide/networking/configure-dns.md

```diff
@@ -0,0 +1,118 @@
+<!--[metadata]>
++++
+title = "Configure container DNS in user-defined networks"
+description = "Learn how to configure DNS in user-defined networks"
+keywords = ["docker, DNS, network"]
+[menu.main]
+parent = "smn_networking"
++++
+<![end-metadata]-->
+
+# Embedded DNS server in user-defined networks
+
+The information in this section covers the embedded DNS server operation for containers in user-defined networks. DNS lookup for containers connected to user-defined networks works differently compared to the containers connected to `default bridge` network.
```

Comment:
Should we mention the differences? i.e., doesn't add entries to `/etc/hosts`, and doesn't create environment-variables?